core/src/main/java/org/lflang/generator/docker/DockerComposeGenerator.java

```diff
@@ -116,7 +100,53 @@ public void writeDockerComposeFile(List<DockerData> services, String networkName
     var contents =
         String.join(
             "\n", this.generateDockerServices(services), this.generateDockerNetwork(networkName));
-    FileUtil.writeToFile(contents, path);
-    context.getErrorReporter().nowhere().info(getUsageInstructions());
+    FileUtil.writeToFile(
+        contents, context.getFileConfig().getSrcGenPath().resolve("docker-compose.yml"));
+  }
+
+  /**
+   * Build using docker compose.
+   *
+   * @return {@code true} if successful,{@code false} otherwise.
+   */
+  public boolean build() {
+    return Objects.requireNonNull(
+                LFCommand.get(
+                    "docker",
+                    List.of("compose", "build"),
+                    false,
+                    context.getFileConfig().getSrcGenPath()))
+            .run()
+        == 0;
+  }
+
+  /** Create a launcher script that invokes Docker. */
+  public void createLauncher() {
+    var fileConfig = context.getFileConfig();
+    var packageRoot = fileConfig.srcPkgPath;
+    var srcGenPath = fileConfig.getSrcGenPath();
+    var file = fileConfig.binPath.resolve(fileConfig.name).toFile();
+    var script =
+        """
+        #!/bin/sh
```

Comment:
```suggestion
        #!/bin/bash
```

`sh` will fail with the error `set: Illegal option -o pipefail`

Reply:
Nice catch!